sym_ewram.txt

```diff
@@ -235,203 +171,204 @@ gUnknown_202D648: /* 202D648 */
     .include "src/bg_control.o"
 	.space 0x140
 
-gBldAlpha: /* 202D7FA */
+gBldAlpha: /* 202D7FA (bg_control / cpu) */
 	.space 0x2
 
-gBldCnt: /* 202D7FC */
+gBldCnt: /* 202D7FC (bg_control / cpu / main / reg_control) */
 	.space 0x2
 
-gUnknown_202D7FE: /* 202D7FE */
+gUnknown_202D7FE: /* 202D7FE (bg_control / main / reg_control) */
 	.space 0x2
 
-gUnknown_202D800: /* 202D800 */
+gUnknown_202D800: /* 202D800 (cpu) */
 	.space 0x8
 
-gUnknown_202D808: /* 202D808 */
+gUnknown_202D808: /* 202D808 (code_800D090) */
 	.space 0x108
 
-gUnknown_202D910: /* 202D910 */
+gUnknown_202D910: /* 202D910 (code_800D090) */
 	.space 0x4
 
-gUnknown_202D914: /* 202D914 */
+gUnknown_202D914: /* 202D914 (code_800D090) */
 	.space 0x20C
 
-gUnknown_202DB20: /* 202DB20 */
+gUnknown_202DB20: /* 202DB20 (code_800D090) */
 	.space 0x10
 
-gUnknown_202DB30: /* 202DB30 */
+gUnknown_202DB30: /* 202DB30 (code_800D090) */
 	.space 0x20
 
-gUnknown_202DB50: /* 202DB50 */
+gUnknown_202DB50: /* 202DB50 (code_800D090) */
 	.space 0x8
 
-gUnknown_202DB58: /* 202DB58 */
+gUnknown_202DB58: /* 202DB58 (code_800D090) */
 	.space 0x8
 
-gUnknown_202DB60: /* 202DB60 */
+gUnknown_202DB60: /* 202DB60 (code_800D090) */
 	.space 0x8
 
-gUnknown_202DB68: /* 202DB68 */
+gUnknown_202DB68: /* 202DB68 (code_800D090) */
 	.space 0x4
 
-gUnknown_202DB6C: /* 202DB6C */
+gUnknown_202DB6C: /* 202DB6C (code_800D090) */
 	.space 0x4
 
-gUnknown_202DB70: /* 202DB70 */
+gUnknown_202DB70: /* 202DB70 (code_800D090) */
 	.space 0x4
 
-gUnknown_202DB74: /* 202DB74 */
+gUnknown_202DB74: /* 202DB74 (code_800D090) */
 	.space 0x4
 
-gUnknown_202DB78: /* 202DB78 */
+gUnknown_202DB78: /* 202DB78 (code_800D090) */
 	.space 0x40
 
-gUnknown_202DBB8: /* 202DBB8 */
+gUnknown_202DBB8: /* 202DBB8 (code_800D090 / data_80B9BB8) */
 	.space 0x140
 
-gUnknown_202DCF8: /* 202DCF8 */
+gUnknown_202DCF8: /* 202DCF8 (code_800D090 / arm_func) */
 	.space 0x120
 
-gUnknown_202DE18: /* 202DE18 */
+gUnknown_202DE18: /* 202DE18 (code_800DAC0) */
 	.space 0x1
 
-gUnknown_202DE19: /* 202DE19 */
+gUnknown_202DE19: /* 202DE19 (code_800DAC0) */
 	.space 0x3
 
-gUnknown_202DE1C: /* 202DE1C */
+gUnknown_202DE1C: /* 202DE1C (code_80118A4) */
 	.space 0x4
 
-gUnknown_202DE20: /* 202DE20 */
+gUnknown_202DE20: /* 202DE20 (code_80118A4) */
 	.space 0x2
 
-gUnknown_202DE22: /* 202DE22 */
+gUnknown_202DE22: /* 202DE22 (code_80118A4) */
 	.space 0x2
 
-gUnknown_202DE24: /* 202DE24 */
+gUnknown_202DE24: /* 202DE24 (code_80118A4) */
 	.space 0x4
 
-gUnknown_202DE28: /* 202DE28 */
+gUnknown_202DE28: /* 202DE28 (save) */
 	.space 0x4
 	.align 4
 
 gUnknown_202DE30: /* 202DE30 */
 	.space 0x28
 
+/* Seems to be a general u8 buffer for text, mostly used with item names */
 gUnknown_202DE58: /* 202DE58 */
-	.space 0x50
+	.space 80
```

Comment:
Why are a handful of these changed to decimal instead of hex?

Reply:
I changed it only where it's appropriate. Some of those buffers are 80/160/240 bytes long, so showing the 80/160 makes more sense at a glance than 0x50/0xA0 imo